.clangd

```diff
@@ -0,0 +1,2 @@
+CompileFlags:
+  CompilationDatabase: third_party/p4c/build/
```

Comment:
This tied to the way how the things are built on CI, but effectively disallows e.g. just to drop stuff into `extensions` of existing p4c. So I'd rather not include here as this depends on the particular build directory layout.

Reply:
`third_part/p4c` only exists for the purpose of CI? I assumed it is meant to be a standalone build.

Reply:
It works just fine compiling as a standalone extension, so I would explicitly not to disallow this possibility (and actually this is how I'm compiling the things locally).

Reply:
Iirc the compilation database here is just a hint, you could also pass it to your IDE options of your choice. But this gives a nicer out-of-the-box experience. Let me check whether multiple folders are possible.